doc/how-to/getting_started_db.rst

```diff
@@ -1,13 +1,282 @@
+        :dedent:
+
+#.  Select tuples using the ``secondary`` index:
+
+    ..  literalinclude:: /code_snippets/snippets/config/instances.enabled/create_db/data.lua
+        :language: lua
+        :lines: 28
+        :dedent:
+
+#.  To prepare for the example in the next section, grant read, write, and execute
+    privileges to the current user:
+
+    ..  literalinclude:: /code_snippets/snippets/config/instances.enabled/create_db/data.lua
+        :language: lua
+        :lines: 10
+        :dedent:
+
+The full example in the terminal looks like this:
+
+..  code-block:: tarantoolsession
+
+    create_db:instance001> box.schema.space.create('bands')
+    ---
+    ...
+    create_db:instance001> box.space.bands:format({
+                         > {name = 'id', type = 'unsigned'},
+                         > {name = 'band_name', type = 'string'},
+                         > {name = 'year', type = 'unsigned'}
+                         > })
+    ---
+    ...
+    create_db:instance001> box.space.bands:create_index('primary', { parts = { 'id' } })
+    ---
+    - unique: true
+      parts:
+      - type: unsigned
+        is_nullable: false
+        fieldno: 1
+      id: 0
+      space_id: 512
+      name: primary
+      type: TREE
+    ...
+    create_db:instance001> box.space.bands:insert { 1, 'Roxette', 1986 }
+    ---
+    - [1, 'Roxette', 1986]
+    ...
+    create_db:instance001> box.space.bands:insert { 2, 'Scorpions', 1965 }
+    ---
+    - [2, 'Scorpions', 1965]
+    ...
+    create_db:instance001> box.space.bands:insert { 3, 'Ace of Base', 1987 }
+    ---
+    - [3, 'Ace of Base', 1987]
+    ...
+    create_db:instance001> box.space.bands:select { 3 }
+    ---
+    - - [3, 'Ace of Base', 1987]
+    ...
+    create_db:instance001> box.space.bands:create_index('secondary', { parts = { 'band_name' } })
+    ---
+    - unique: true
+      parts:
+      - fieldno: 2
+        sort_order: asc
+        type: string
+        exclude_null: false
+        is_nullable: false
+      hint: true
+      id: 1
+      type: TREE
+      space_id: 512
+      name: secondary
+    ...
+    create_db:instance001> s.index.secondary:select{'Scorpions'}
+    ---
+    - - [2, 'Scorpions', 1965]
+    ...
+    create_db:instance001> box.schema.user.grant('guest', 'read,write,execute', 'universe')
+    ---
+    ...
+
+..  _connecting-remotely:
+
+Connecting remotely
+-------------------
+
+In the configuration file (``config.yaml``), the instance listens on ``127.0.0.1:3301``:
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/create_db/config.yaml
+    :language: yaml
+    :lines: 6-8
+    :dedent:
+
+The ``listen`` value can be any form of a :ref:`URI <index-uri>` (uniform resource identifier).
+You can send requests to the listen URI using:
+
+*   ``telnet``
+*   a :ref:`connector <index-box_connectors>`
+*   another instance of Tarantool (using the :ref:`console <console-module>` module)
+*   :ref:`tt <tt-cli>` administrative utility
```

Comment:
I guess it is for a control (console) socket. `iproto.listen` is about a binary protocol socket. You can't work with it using telnet.

OTOH, you can't use a connector to connect to a console (not a binary protocol socket).

Maybe we should describe it in a more general context, not just `iproto.listen`?